spec/namespaces/notifications.yaml

```diff
@@ -4,6 +4,17 @@ info:
   description: OpenSearch Notifications API
   version: 1.0.0
 paths:
+  /_plugins/_notifications/channels:
+    get:
+      operationId: notifications.list_channels.0
+      x-operation-group: notifications.list_channels
+      x-version-added: '2.0'
+      description: List created notification channels.
+      externalDocs:
+        url: https://opensearch.org/docs/latest/observing-your-data/notifications/api/#list-all-notification-configurations
```

Comment:
there is no good link currently on the docs, I'll try to raise a request there to add one

Reply:
https://github.com/opensearch-project/documentation-website/pull/6982

Reply:
Thank you. Let's add a `TODO` comment saying that the URL needs to be updated once we have a proper URL.

Reply:
I hope it will be merged soon, but added the TODO for now.